README.md

```
@@ -947,5 +945,48 @@ These custom gradle tasks are the following:</br>
 
+# Troubleshooting
+
+## java.io.FileNotFoundException (File name too long)
+
+`java.io.FileNotFoundException: ... (File name too long)`
+
+This might be due to the `AndroidPreviewScreenshotIdBuilder`, try commenting it out and trying again.
+
+## Cannot inline bytecode built with JVM target 17
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)